### ISSUE TITLE
Should NSI contain specific VAT id?

### ISSUE THREAD

**Issue by LaoshuBaby** [osmlab/name-suggestion-index#6802]
I noticed the key `ref:vatin` in taginfo and noticed this project using it. It only appear once in this :

https://github.com/osmlab/name-suggestion-index/blob/6e93a1c62edb33aea1b378e466711c87f327b4ed/dist/taginfo.json#L824-L837

From:

https://github.com/osmlab/name-suggestion-index/blob/0668923e60ebf3ec13b908ab6512cb036dee497c/data/brands/amenity/fast_food.json#L9008-L9024

Should it be seen as website/phone and move to Wikidata?

**Comment by bhousel** [comment]
I actually don't know how these `ref:vatin` values work - but if the value is true for _every_ location of this business, I guess it is ok to include here.. 

Looking at the Burger King wikidata [Q177054](https://www.wikidata.org/wiki/Q177054), I don't see an option to add Taiwan VAT.  Maybe that would change in the future - there are plenty of other identifiers supported - but for now I guess we can capture this in NSI if the mappers want it on every Taiwan Burger King.

**Comment by LaoshuBaby** [comment]
> but if the value is true for _every_ location of this business, I guess it is ok to include here..

Yes, I meet with a similar issue `ref:isil` some days ago: https://github.com/osmlab/name-suggestion-index/pull/6648#discussion_r889668255

(That's why I also show it in this question issue, I'll ask Taiwan community for how this [`ref:vatin`](https://wiki.openstreetmap.org/wiki/Key%3Aref%3Avatin) in Burger King work)

I'll remove it only when this VATIN is unproper. 😆
